### PR TITLE
Adds Assist action documentation

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Adds Assist action documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/16829
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/dashboards/actions.markdown [hunk 1]
```diff
@@ -61,6 +61,16 @@ tap_action:
       description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
       type: [boolean, map]
       default: "false"
+    pipeline_id:
+      required: false
+      description: "Assist pipeline id to use when `action` defined as `assist`"
```
Comment (c0ffeeca7) [inline source/dashboards/actions.markdown]:
```suggestion
      description: "Assist pipeline id to use when the `action` is defined as `assist`"
```

File: source/dashboards/actions.markdown [hunk 2]
```diff
@@ -61,6 +61,16 @@ tap_action:
...
+      type: string
+      default: none
+    start_listening:
+      required: false
+      description: "If supported, listen for voice commands when opening the assist dialog when `action` defined as `assist`"
```
Comment (c0ffeeca7) [inline source/dashboards/actions.markdown]:
```suggestion
      description: "If supported, listen for voice commands when opening the assist dialog and the `action` is defined as `assist`".
```

File: source/dashboards/actions.markdown [hunk 3]
```diff
@@ -108,6 +118,16 @@ hold_action:
       description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
       type: [boolean, map]
       default: "false"
+    pipeline_id:
+      required: false
+      description: "Assist pipeline id to use when `action` defined as `assist`"
```
Comment (c0ffeeca7) [inline source/dashboards/actions.markdown]:
```suggestion
      description: "Assist pipeline id to use when the `action` is defined as `assist`"
```

File: source/dashboards/actions.markdown [hunk 4]
```diff
@@ -108,6 +118,16 @@ hold_action:
...
+      type: string
+      default: none
+    start_listening:
+      required: false
+      description: "If supported, listen for voice commands when opening the assist dialog when `action` defined as `assist`"
```
Comment (c0ffeeca7) [inline source/dashboards/actions.markdown]:
```suggestion
      description: "If supported, listen for voice commands when opening the assist dialog and the `action` is defined as `assist`"
```

File: source/dashboards/actions.markdown [hunk 5]
```diff
@@ -155,6 +175,16 @@ double_tap_action:
       description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
       type: [boolean, map]
       default: "false"
+    pipeline_id:
+      required: false
+      description: "Assist pipeline id to use when `action` defined as `assist`"
```
Comment (c0ffeeca7) [inline source/dashboards/actions.markdown]:
```suggestion
      description: "Assist pipeline id to use when the `action` is defined as `assist`"
```

File: source/dashboards/actions.markdown [hunk 6]
```diff
@@ -155,6 +175,16 @@ double_tap_action:
...
+      type: string
+      default: none
+    start_listening:
+      required: false
+      description: "If supported, listen for voice commands when opening the assist dialog when `action` defined as `assist`"
```
Comment (c0ffeeca7) [inline source/dashboards/actions.markdown]:
```suggestion
      description: "If supported, listen for voice commands when opening the assist dialog when `action` defined as `assist`"
```
```suggestion
      description: "If supported, listen for voice commands when opening the assist dialog and the `action` is defined as `assist`"
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
tiny tweaks

**Comment by piitaya** [comment]
Thank you for feedback @c0ffeeca7 !

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @piitaya :+1:

**Pull request merged**